<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>870</width>
         <height>506</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelMain">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="action_Zoom"/>
    <addaction name="action_Fit_to_Windows"/>
    <addaction name="action_Show_Dialog"/>
    <addaction name="action_DarkMode"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="action_Deutsch"/>
    <addaction name="action_Englisch"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Language"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_as"/>
   <addaction name="action_Print"/>
   <addaction name="actionE_xit"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="action_Zoom"/>
   <addaction name="action_Fit_to_Windows"/>
   <addaction name="action_Show_Dialog"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
   <addaction name="actionAbout_Qt"/>
   <addaction name="action_Deutsch"/>
   <addaction name="action_Englisch"/>
   <addaction name="action_DarkMode"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="statusTip">
    <string>Open a new image</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;as</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/fileprint.png</normaloff>:/images/images/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/copy.png</normaloff>:/images/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/paste.png</normaloff>:/images/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/zoomin.png</normaloff>:/images/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/zoomout.png</normaloff>:/images/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="action_Zoom">
   <property name="text">
    <string>&amp;Zoom 100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="action_Fit_to_Windows">
   <property name="icon">
    <iconset resource="ressources_icons.qrc">
     <normaloff>:/images/images/fittoview.png</normaloff>:/images/images/fittoview.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fit to Windows</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Show_Dialog">
   <property name="text">
    <string>&amp;Show Dialog</string>
   </property>
  </action>
  <action name="action_Deutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Deutsch</string>
   </property>
  </action>
  <action name="action_Englisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="action_DarkMode">
   <property name="text">
    <string>DarkMode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources_icons.qrc"/>
 </resources>
 <connections/>
</ui>
